bigtable_automation/gcf/prod.go

```diff
@@ -72,7 +72,7 @@ func prodInternal(ctx context.Context, e GCSEvent) error {
 	// Get table ID.
 	// e.Name should is like "**/*/branch_2021_01_01_01_01/launched.txt"
 	parts := strings.Split(e.Name, "/")
-	if len(parts) < 3 {
+	if len(parts) < 2 {
```

Comment:
Local test script sends '<tablename>/init.txt', which only has two parts? It the 3+ parts intentional? (test can be changed to add something random before <tablename>.

Reply:
What is local test? I used to deploy this to the dev instance so this might not account for it?

Reply:
Local test is from README, linked below. Specifically, line 50  is "'"$CACHE_NAME"'/init.txt", and CACHE_NAME is a single part, so 2 parts. Currently local test doesn't work?

https://github.com/datacommonsorg/tools/blob/1dec790d08fdbad8f6f1d2e5d9a05beced907094/bigtable_automation/gcf/local/test.sh#L50

Reply:
If changing it to 2, then in GCP, some unwanted change will not be filtered out. Maybe update the local name here to make it at least 3 segments?

Reply:
> some unwanted change will not be filtered out

Makes sense. Changed local test to be like branch cache, keeping 3+ segments